packages/quantic/force-app/main/default/lwc/quanticFeedbackModal/feedbackForm.html

```diff
@@ -0,0 +1,36 @@
+<template>
+  <template if:true={validated}>
+    <lightning-modal-header label={labels.explainWhy}></lightning-modal-header>
+    <lightning-modal-body>
+      <div class="slds-var-p-around_x-large">
+        <lightning-radio-group class="slds-var-m-bottom_large" name="feedback" label={labels.selectTheReason}
+          options={options} value={feedbackValue} type="radio" onchange={handleOptionChange}></lightning-radio-group>
+        <div class="details-input__container">
+          <template if:true={displayDetailsInput}>
+            <div class={detailsFormClass}>
+              <label class="slds-form-element__label details-input__label" for="details-input">
+                {labels.whatWasTheReason}</label>
+              <div class="slds-form-element__control">
+                <textarea rows="4" id="details-input" aria-describedby="details-error" class="slds-textarea"
```

Comment:
Are you sure about the `aria-describedby` `details-error` here?
Seems like a described by should be used to describe the purpose of the field/input, not described by the error message?

Reply:
I think it might make sense following this page:
https://www.w3.org/WAI/tutorials/forms/notifications/

But also there is the pattern under : `Binary messages`
If the error message "appears" while you're typing or after an action and is not already visible in the page, you will need to use a `aria-live` to tell screen readers that a new message has appeared in the page.

Reply:
I updated this to directly use the `lightning-textarea` component, which handles this a11y concerns for us.